test/fixtures/roles.sql

```diff
@@ -7,46 +7,50 @@ CREATE ROLE postgrest_test_author;
 GRANT postgrest_test_anonymous, postgrest_test_default_role, postgrest_test_author TO :USER;
 
 -- reloadable config options for io tests
-ALTER ROLE postgrest_test_authenticator SET pgrst."jwt-aud" = 'https://example.org';
-ALTER ROLE postgrest_test_authenticator SET pgrst."openapi-server-proxy-uri" = 'https://example.org/api';
-ALTER ROLE postgrest_test_authenticator SET pgrst."raw-media-types" = 'application/vnd.pgrst.db-config';
-ALTER ROLE postgrest_test_authenticator SET pgrst."jwt-secret" = 'REALLYREALLYREALLYREALLYVERYSAFE';
-ALTER ROLE postgrest_test_authenticator SET pgrst."jwt-secret-is-base64" = 'true';
-ALTER ROLE postgrest_test_authenticator SET pgrst."jwt-role-claim-key" = '."a"."role"';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-tx-end" = 'commit-allow-override';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-schemas" = 'test, tenant1, tenant2';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-root-spec" = 'root';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-prepared-statements" = 'false';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-pre-request" = 'test.custom_headers';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-max-rows" = '1000';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-extra-search-path" = 'public, extensions';
+ALTER ROLE postgrest_test_authenticator SET pgrst."jwt_aud" = 'https://example.org';
+ALTER ROLE postgrest_test_authenticator SET pgrst."openapi_server_proxy_uri" = 'https://example.org/api';
+ALTER ROLE postgrest_test_authenticator SET pgrst."raw_media_types" = 'application/vnd.pgrst.db-config';
+ALTER ROLE postgrest_test_authenticator SET pgrst."jwt_secret" = 'REALLYREALLYREALLYREALLYVERYSAFE';
+ALTER ROLE postgrest_test_authenticator SET pgrst."jwt_secret_is_base64" = 'true';
+ALTER ROLE postgrest_test_authenticator SET pgrst."jwt_role_claim_key" = '."a"."role"';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_tx_end" = 'commit-allow-override';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_schemas" = 'test, tenant1, tenant2';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_root_spec" = 'root';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_prepared_statements" = 'false';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_pre_request" = 'test.custom_headers';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_max_rows" = '1000';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_extra_search_path" = 'public, extensions';
+
+-- override with database specific setting
+ALTER ROLE postgrest_test_authenticator SET pgrst."db_max_rows" = '1111';
```

Comment:
Maybe we can do something like this, too?

```suggestion
DROP DATABASE IF EXISTS other;
CREATE DATABASE other;
ALTER ROLE postgrest_test_authenticator IN DATABASE other SET pgrst."db_max_rows" = '1111';
```

And then test that `db-max-rows` is **not** 1111, but still 1000.

Reply:
I though this couldn't be done because of `CREATE DATABASE cannot be executed inside a transaction block`. But I've tried and it works! We only got `\set ON_ERROR_STOP on` set for the scripts.